### DISCUSSION TITLE
Combat Diving Suit propulsion multiplier swapped

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#13825] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

combat diving suit's propulsion speed multiplier was lost and replaced with the regular speed multiplier's value. 

I believe this to be a bug because the 1.4 multiplier made the suit agile

### Reproduction steps

- SpeedMultiplier="0.95" PropulsionSpeedMultiplier="1.4"
+ PropulsionSpeedMultiplier="0.95" 

in content/items/divinggear.xml line 450

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for your report, this was an oversight and will be addressed!
